examples/classification.py

```diff
@@ -16,31 +16,31 @@
 )
 
 FNN = Sequential(
-    Linear(784, 128),
+    Linear(784, 32),
```

Comment:
Any reason to change to 32 instead of 128?

Reply:
I think we need to debug with smaller networks what is the issue with Batchnorm. I put back the FNN to 128, but left the FNN_BATCHNORM & FNN_LAYERNORM to 128.